### ISSUE TITLE
Sortable lists should be more obviously sortable

### ISSUE THREAD

**Issue by jagthedrummer** [bullet-train-co/bullet_train-core#826]
Currently if you super scaffold a sortable model you'll end up with an index page that is technically sortable, but there's no indication given to the user that the items can be reordered.

```
rails g super_scaffold Blog Team name:text_field --sortable
```

![CleanShot 2024-05-07 at 11 10 14](https://github.com/bullet-train-co/bullet_train-core/assets/58702/f13b1580-1c8d-4c1c-b55e-6c2119a850a3)

I think we should probably add a defined "drag handle" (something like ↕️) to help indicate that the items are draggable.

It would also be nice to change the cursor when someone is hovered over the draggable item. Probably `cursor: grab` when hovered over the item, and then `cursor: grabbing` when the drag has been started.

The lack of dragging indicators might be at play in this comment: https://github.com/bullet-train-co/bullet_train-core/pull/825#issuecomment-2098188135

**Comment by swombat** [comment]
Agreed. I ended up disabling on most lists this since it made clicking less reliable, and gave rise to users saying "the list isn't working" because they moved the mouse a tiny bit while clicking.

I'd advise:
- drag handle
- light dashed outline

Between those two things it should be more obvious that this is draggable.

**Comment by swombat** [comment]
This from the Refactoring UI book suggests an even better solution, visually:

<img width="824" alt="image" src="https://github.com/bullet-train-co/bullet_train-core/assets/4488/c7d2204d-98b9-4422-941a-7d41d1aad7a8">
